widget/ui/src/theme.ts

```diff
@@ -228,3 +225,16 @@ export const darkTheme = createTheme('dark-theme-ui', {
     s: '0px 3px 5px 3px #222, 0px 6px 10px 3px #222, 0px 1px 18px 3px #222',
   },
 });
+
+export const RangoDarkColors = {
```

Comment:
Please follow other naming convention in this file. it should be `rangoDarkColors`